src/ucm/configure.m4

```diff
@@ -4,8 +4,13 @@
 # See file LICENSE for terms.
 #
 
-AC_SUBST([UCM_MODULE_LDFLAGS],
-         ["-Xlinker -z -Xlinker interpose -Xlinker --no-as-needed"])
+AS_CASE([$host_os],
+    [linux*], [
+        AC_SUBST([UCM_MODULE_LDFLAGS],
+                 ["-Xlinker -z -Xlinker interpose -Xlinker --no-as-needed"])
+    ],
+    [AC_MSG_ERROR([UCM_MODULE_LDFLAGS=PORT ME])]
+)
```

Comment:
need to test for linker flags support, potentially by AC_LINK_IFELSE

Reply:
I added AC_LINK_IFELSE test.